File: 03-sum-nat/sum_nat.py
# Python program to sum numbers from 1 to n
# Created by M.V.Harish Kumar on 31-10-2023

def sum_to_n(n):
    sum = 0
    for i in range(n+1):
        sum += i
    return sum

n = int(input("Enter value for `n`: "))
ans = sum_to_n(n)
print(f"The sum from 1 to {n} is {ans}")

File: 03-sum-nat/sum_nat.pl
# Perl program to sum numbers from 1 to n
# Created by M.V.Harish Kumar on 09-01-2025
use v5.40;

sub sum_to_n($n) {
	my $sum = 0;
	$sum += $_ for 1..$n;
	return $sum;
}

print "Enter value for `n`: ";
chomp(my $n = <STDIN>);
my $ans = sum_to_n $n;
say "The sum from 1 to $n is $ans";
